index.html

```diff
@@ -1978,12 +1978,14 @@ <h3>Terms of Use</h3>
         <h3>Evidence</h3>
 
         <p>
-Evidence is used by an <a>issuer</a> to determine whether or not to issue
-a <a>credential</a>. For example, an issuer might check physical documentation
-provided by the <a>subject</a> or might perform a set of background checks
-before issuing the credential. In certain scenarios, this information is useful
-to the <a>verifier</a> when determining the risk associated with relying on a
-given credential.
+Evidence is used by an <a>issuer</a> to inform the <a>verifier</a> of the basis
```

Comment:
Evidence is used by an <a>issuer</a> to inform the <a>verifier</a> of the basis
for issuing the <a>verifiable credential</a>. This could be used by the <a>verifier</a>